_posts/2020-03-16-dask-usage-at-blue-yonder.markdown

```diff
@@ -0,0 +1,161 @@
+---
```

Comment:
I know I'm a markdown Nazi, but I personally _really_ like to rely on markdown's formatter, i.e. one sentence = one line. That's IMHO much easier to review/comment then mixed sentences in one line.

```
This is a sentence.
This is another sentence.
And this is how markdown renders it.
```

This is a sentence. This is another sentence. And this is how Markdown renders it if everything is configured correctly.

Reply:
Why not a paragraph per line? Then, what you see is what you get and it makes inserting suggestions easier

Reply:
IMHO then it's pretty hard to review.

The changelog might look like 60 lines for adding a single `,`.

Reply:
Or if you don't like to do sentences, then at least use a reasonable line length (like 120chars).

Reply:
I think that's what I did. For longer sentences, the lines get longer than I like otherwise. But it might be a good idea to start each new sentence on a new line.